apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -922,6 +1038,13 @@ private String getSummary(final String preferencesKey, final String defaultValue
             void setEnabled(boolean enabled);
         }
 
+        private interface IsEnabled {
```

Comment:
The semantic is wrong here and all the local variables.
It's not a boolean or an enabled state, please rename.